PR: https://github.com/odoo/odoo/pull/

From: 67e148c20f82de862b3f78fc5a54d1e63555600e
From: Joseph Caburnay
Date: 2020-06-26 05:40:23

Structural Changes: 5
Total Changes: 6

[IMP] point_of_sale: various pos backend views and pos ui improvements

TASK-ID: 2230195

================================= pseudo patch: =================================

--- a/addons/point_of_sale/models/pos_config.py
+++ b/addons/point_of_sale/models/pos_config.py
@@ -194,8 +194,8 @@ class PosConfig(models.Model):
     use_pricelist = fields.Boolean("Use a pricelist.")
     tax_regime = fields.Boolean("Tax Regime")
     tax_regime_selection = fields.Boolean("Tax Regime Selection value")
-    start_category = fields.Boolean("Set Start Category", default=False)
-    limit_categories = fields.Boolean("Restrict Available Product Categories")
+    start_category = fields.Boolean("Start Category", default=False)
+    limit_categories = fields.Boolean("Restrict Product Categories")
     module_account = fields.Boolean(string='Invoicing', help='Enables invoice generation from the Point of Sale.')
     module_pos_restaurant = fields.Boolean("Is a Bar/Restaurant")
     module_pos_discount = fields.Boolean("Global Discounts")

--- a/addons/point_of_sale/models/pos_session.py
+++ b/addons/point_of_sale/models/pos_session.py
@@ -32,7 +32,7 @@ class PosSession(models.Model):
         index=True)
     name = fields.Char(string='Session ID', required=True, readonly=True, default='/')
     user_id = fields.Many2one(
-        'res.users', string='Responsible',
+        'res.users', string='Opened By',
         required=True,
         index=True,
         readonly=True,
